content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -344,10 +344,17 @@ We will call the `helloworld.sample` service from another in-mesh `sleep` servic
     $ kubectl create --context=$CTX_CLUSTER1 -f @samples/sleep/sleep.yaml@ -n sample
     {{< /text >}}
 
+1. Wait for the `sleep` service to start:
+
+    {{< text bash >}}
+    $ kubectl get po --context=$CTX_CLUSTER1 -n sample -l app=sleep
+    sleep-754684654f-n6bzf           2/2     Running   0          5s
+    {{< /text >}}
+
 1. Call the `helloworld.sample` service several times:
 
     {{< text bash >}}
-    $ kubectl exec --context=$CTX_CLUSTER1 -it -n sample -c sleep $(kubectl get pod --context=$CTX_CLUSTER1 -n sample -l app=sleep -o jsonpath={.items[0].metadata.name}) -- curl helloworld.sample:5000/hello
+    $ kubectl exec --context=$CTX_CLUSTER1 -it -n sample -c sleep $(kubectl get pod --context=$CTX_CLUSTER1 -n sample -l app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl helloworld.sample:5000/hello
```

Comment:
This seems inconsistent with the previous ones. There you put `'` around the whole argument value while here you're just doing the `=value` part of the `-o` argument.

Reply:
I was trying to be consistent with the other files which added ' or "s in their -o jsonpath=... (ex: content/docs/examples/advanced-gateways/egress-gateway-tls-origination/index.md)

Reply:
I think that we should fix all of the z shell problem for all of the docs in a consistent way and in one PR.

Reply:
Commit #2 in this PR should handle the z shell problem constantly across all docs. Standardized on single quoting on one side of the `=` unless single quotes were already used, then left the double-quote.